helm/cluster-vsphere/values.schema.json

```diff
@@ -762,6 +881,20 @@
             "type": "object",
             "additionalProperties": false,
             "properties": {
+                "advancedConfiguration": {
+                    "type": "object",
+                    "title": "Advanced configuration",
+                    "description": "Advanced configuration options passed to shared cluster chart.",
+                    "default": {
+                        "controlPlane": {
+                            "apiServer": {
+                                "extraArgs": {
+                                    "requestheader-allowed-names": "front-proxy-client"
```

Comment:
we're not entirely sure why this is in the vsphere chart - it may be possible to drop it but as we're not sure i've elected to keep it for now.